content/docs/1.9.0/v2-data-engine/features/ublk-frontend-support.md

```diff
@@ -0,0 +1,70 @@
+---
+title: UBLK Frontend Support
+weight: 20
+aliases:
+- /spdk/features/ublk-frontend-support.md
+---
+
+Starting with v1.9.0, Longhorn supports UBLK frontend for v2 data engine volume.
+This feature exposes v2 data engine volume as a block device by using [UBLK SPDK framework](https://spdk.io/doc/ublk.html).
+In a very high-spec environment, the UBLK frontend might have better performance than the default NVMe-oF frontend of v2 data engine volume.
```

Comment:
We might want to be more specific about what `very high-spec environment` means.

Reply:
Thank you. Fixed